### ISSUE TITLE
Twit Live

### ISSUE THREAD

**Issue by EllimanMC** [plexinc-plugins/TWiT.bundle#12]
Is live streaming still supported?  I receive error: "Couldn't create the playback session for this item."  Recorded shows play without issue.

**Comment by sander1** [comment]
Yes, it is, except in the web client. The web client can't play videos that are in the HTTP Live Stream format (which is what TWiT is using). PMS cannot transcode live streams, so the only way to watch this type of streams is with a Plex client that can play those streams natively (like Roku, iOS, Android and many more Plex clients, except web based clients).

**Comment by EllimanMC** [comment]
Interesting ... My tvOS / iOS plex clients cannot play the live stream.

**Comment by sander1** [comment]
It's working ok on iOS for me...

![img_3596](https://cloud.githubusercontent.com/assets/104684/24470393/12cb3f9e-14bf-11e7-9484-c207e472d542.jpg)

I just noticed that the channel preferences still contained old options for the livestream. I've removed those and pushed out an update. 

Make sure you're using "Flosoft.biz" as livestream source:

![screen shot 2017-03-29 at 20 38 13](https://cloud.githubusercontent.com/assets/104684/24470599/cba8f538-14bf-11e7-81f0-d3a9ab90f2c5.jpg)

![screen shot 2017-03-29 at 20 38 18](https://cloud.githubusercontent.com/assets/104684/24470604/ce667f48-14bf-11e7-8d0e-f0654a6ca248.png)

**Comment by EllimanMC** [comment]
Thanks,  i see the update and changed the source to flosoft.biz.  Oddly enough its working on my android device.  iOS / tvOS still not working.

**Comment by EllimanMC** [comment]
Any other thoughts as to what could be causing my iOS/tvOS issue?
